Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -473,6 +439,17 @@ Publish-WebSite $zipPackage $appDirectory $appName $overwriteWebConfig
 Write-Host ""
 Add-DatabaseSecurity $iisUser $identityDatabaseRole $identityDbConnStr
 
+if(!($noDiscoveryService)){
+    $buildVersion = [System.Diagnostics.FileVersionInfo]::GetVersionInfo("$appDirectory\Fabric.Identity.API.dll").FileVersion
+    $serviceName = "IdentityService"
+    $serviceVersion = 1
+    $friendlyName = "Fabric.Identity"
+    $description = "The Fabric.Identity service provides centralized authentication across the Fabric ecosystem."
+    Add-ServiceUserToDiscovery $credential.UserName $metadataConnStr
+    Add-DiscoveryRegistration $discoveryServiceUrl $identityServerUrl $credential, $buildVersion, $serviceName, $serviceVersion, $friendlyName, $description
```

Comment:
Should we encapsulate all the elements that make up the body of the post into an object, e.g.:

```
$discoveryPostBody = @{
                                            buildVersion = [System.Diagnostics.FileVersionInfo]::GetVersionInfo("$appDirectory\Fabric.Identity.API.dll").FileVersion
                                            serviceName = "IdentityService"
                                            ...
                                    }
```

It might make this call more readable and easier to work with.